pallets/order-book/src/lib.rs

```diff
@@ -692,7 +692,7 @@ pub mod pallet {
                 // schedule alignment
                 <AlignmentCursor<T>>::set(order_book_id, Some(T::OrderId::zero()));
             }
-            <OrderBooks<T>>::set(order_book_id, Some(order_book));
+            <OrderBooks<T>>::insert(order_book_id, order_book);
```

Comment:
I prefer to keep `set`

Reply:
I went with `insert` because the value is always `Some`. Why do you prefer `set`?

Reply:
Yes. As for me `insert` is more suitable for creation, and `set` for updating